<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RaptorStoryPage</class>
    <widget class="QWidget" name="RaptorStoryPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>660</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="_StoryPageLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="_Header" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>50</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>50</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QVBoxLayout" name="_HeaderLayout">
                        <property name="spacing">
                            <number>5</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QLabel" name="_Title">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>32</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>32</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="_Description">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="_Control" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>36</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>36</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="_ControlLayout">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QWidget" name="_TabPanel" native="true">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <layout class="QHBoxLayout" name="_TabPanelLayout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QPushButton" name="_TabPrev">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabAbout">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                            <property name="checked">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabAboutQt">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabFAQ">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabNext">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <spacer name="_ControlSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="_GitHub">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>26</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>26</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QStackedWidget" name="_Body">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="RaptorAboutPage" name="_AboutPage"/>
                    <widget class="RaptorAboutQtPage" name="_AboutQtPage"/>
                    <widget class="RaptorFAQPage" name="_FAQPage"/>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>RaptorAboutPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Story/About/RaptorAboutPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorAboutQtPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Story/AboutQt/RaptorAboutQtPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorFAQPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Story/FAQ/RaptorFAQPage.h</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
